tests/Multiple/ExtraSmallMultipleHTMLFormatterTest.php

```diff
@@ -24,7 +25,7 @@ final class ExtraSmallMultipleHTMLFormatterTest extends TestCase
     protected function setUp(): void
     {
         $this->formatter = new ExtraSmallMultipleHTMLFormatter(new Translator('nl_NL'));
-        CarbonImmutable::setTestNow(CarbonImmutable::create(2021, 5, 3));
+        CalendarSummaryTester::setTestNow(2021, 5, 3);
```

Comment:
This seems to be the only correct call to `CalendarSummaryTester::setTestNow`

Reply:
Indeed, strange that `composer ci` didn't pick it up

Reply:
I think it did not complain because the method setTestNow had no typehints.